content-src/components/Base/Base.jsx

```diff
@@ -157,7 +157,7 @@ export class BaseContent extends React.PureComponent {
     const outerClassName = [
       "outer-wrapper",
       shouldBeFixedToTop && "fixed-to-top",
-      this.state.fixedSearch && "fixed-search"
+      prefs.showSearch && this.state.fixedSearch && "fixed-search"
```

Comment:
Do you need to remove the listeners on a pref change as well for scrolling?

Reply:
well. right now I keep the scroll handlers on all the time. i guess i can optimize that to only listen if `showSearch` is true 🤔

Reply:
oh nevermind that's in the search component

Reply:
Let's leave it as is for now since the code is less risky, maybe we should file as a follow-up